### DISCUSSION TITLE
5.6 社区会议议题

### DISCUSSION THREAD

**Discussion by seeflood** [mosn/layotto#540] [General]
晚8
议题：
### name 和type解耦 @akkw 
https://github.com/mosn/layotto/issues/513

### 跨云部署、在线实验室 @seeflood
交互设计 v0.2 见：
https://github.com/mosn/layotto/issues/471

上次 action: 先按 v0.2 实现，后续再考虑流量治理的事情 @seeflood 
进展: 
https://github.com/mosn/layotto/issues/530
https://github.com/layotto/java-sdk/pull/19

### 跨云部署 -- File API 改造 @wenxuwan 
#510

上次会议action: 
- 提个 PR，支持 s3 API @wenxuwan 

### transaction mesh @stulzq 先看下
https://developer.aliyun.com/article/786297

### CI 改造、容器镜像改造 @Xunzhuo
#483 

上次会议action:  
- [x] pr review、合并 @seeflood 
- [ ] 裁剪分享视频，上传 b站、贴到官方文档里 @zhenjunMa 
- [ ] 合并后重构文档测试、添加可移植性测试 @seeflood 

进展:
https://github.com/mosn/layotto/issues/532
相关事项待认领

### wasm 进展同步（拿 wasm 写组件）
上次会议action: WASI 没提供封装好的socket api，如果我们要做的话，需要把 https://github.com/WebAssembly/WASI/pull/312  这个 pr 里的代码粘贴过来，风险是遇到问题的话很难查。 @rayowang  和 @zhenjunMa  讨论下

### secret API 功能完善 @ZLBer 
- 可能涉及不兼容变更，需要讨论决策
#472 

### 动态配置下发
https://github.com/mosn/layotto/issues/500

上次会议action:  
- 给出一个设计方案 @seeflood 

###  v0.4 发版事项review
https://github.com/mosn/layotto/milestone/2

上次会议action:  
- @rayowang 和 @akkw 认领修复bug的issue
- 修复完成后 release v0.4

## 2.新需求/提案评审
https://github.com/mosn/layotto/issues/539
待认领

## 3. 携程跨云部署分享
https://capa-cloud.github.io/capa.io/blog/2022/01/18/capa-mecha-sdk-of-cloud-application-api/
## 4. 自由讨论
任何想聊的，比如开发遇到的问题

**Comment by nobodyiam** [comment]
It's better to send the meeting link in advance so that people could join freely.

**Reply by seeflood** [reply]
Good point! We can do it next week

**Reply by Xunzhuo** [reply]
How about creating a google calendar, since we have a fixed meeting time?

**Reply by Xunzhuo** [reply]
https://calendar.google.com/calendar/embed?src=to5fq0i23c97t3me000v0hr1o8%40group.calendar.google.com&ctz=Asia%2FSingapore

**Reply by seeflood** [reply]
@Xunzhuo I don't know how to use it, so the chrome will remind me every week? I think it's ok that we use it as a reminder
